.travis.yml

```diff
@@ -1,25 +1,37 @@
-language: c
+language: cpp
```

Comment:
@rouault you are describing the symptoms not the real issue :) The fact that cmake only works on arch with valid c++ compiler is outside the scope of this question. I also fail to understand why `WITH-ASTYLE=ON` imply a valid c++ compiler in the PATH. I do understand that it should have zero impact on the build itself (a bit more dependencies though).